### PR TITLE
Fix #187 issue (comma in keys)

### LINKED ISSUE #187: Unable to delete keys with comma(s)
My keys list:

изображение

I select `Articles.All(rid=11,did=14,page=12)` and press "Delete selected"

Nothing changed, but I recieved:

изображение

`Select All -> Delete selected` broken too with same situation on screen above

Click on key 

изображение

and click on red cross works, with answer:

изображение
(just as planned)

P.S. Keys with `(`, `)` and `,` allowed by redis:

изображение

### PR DESCRIPTION
**Pull request by KarelWintersky**
- Fix #187 issue (comma in keys)
- little cosmetic fixes

### CODE REVIEW

File: index.php [hunk 1]
```diff
@@ -33,7 +34,7 @@
         continue;
       }
 
-      $key = explode($server['seperator'], $key);
+      $key = explode($server['seperator'], $key); //@todo: may be separator ?
```
Comment (erikdubbelboer) [inline index.php:37]: I don't understand this comment?

File: delete.php [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
-
 if (!isset($_POST['post'])) {
   die('Javascript needs to be enabled for you to delete keys.');
 }
 
-
 require_once 'includes/common.inc.php';
 
+global $redis;
+global $server;
```
Comment (erikdubbelboer) [inline delete.php:10]: I don't understand why all these `global` statements are needed? These files aren't included within functions?

**Review by erikdubbelboer — CHANGES_REQUESTED** [review]

File: index.php (hunk 1 above)
Comment (KarelWintersky) [reply index.php:37]: "Separator", not a "sepErator". Incorrect english word.

File: delete.php (hunk 2 above)
Comment (KarelWintersky) [reply delete.php:10]: PHPStorm syntax analyzer recommend use global statement for explicit declaring that variable is global (and declared before)

File: index.php (hunk 1 above)
Comment (KarelWintersky) [reply index.php:37]: It's not a bug ;-)

Comment (erikdubbelboer) [reply index.php:37]: 🙈 too late to fix now I guess, it would break the configs people currently have.

**Comment by erikdubbelboer** [comment]
Thanks! I'll tag a new release in a bit.

**Pull request merged**